<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProductDlg</class>
 <widget class="QDialog" name="NewProductDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加新产品</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SKU Code:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SKUCode">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>基本信息</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="lineEdit_productName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>产品名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_estimateWeight">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>估算重量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_realWeight">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>实际重量</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>采购信息</string>
    </attribute>
    <widget class="QTableView" name="tableView_buy">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>361</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_newBuyItem">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>新增</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_removeBuyItem">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>移除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>产品属性</string>
    </attribute>
    <widget class="QTableView" name="tableView_attrib">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>361</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_newAttribItem">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>新增</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_removeAttribItem">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>移除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>缩略图</string>
    </attribute>
    <widget class="QLabel" name="label_thumb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>351</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_imagePath">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_imagePath">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>店铺</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>备注</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit_Note">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>351</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_newProduct">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
